packages/docs/page-config/styles/smart-helpers/examples/Spacing.vue

```diff
@@ -1,41 +1,41 @@
 <template>
```

Comment:
Seems like doesn't work. 
<img width="622" alt="image" src="https://user-images.githubusercontent.com/29167241/219408235-0ac5aa59-65c7-4d31-847d-4cfa400528a6.png">

Reply:
@aluarius, make sure plugins are registered in nuxt module.